public/assets/data.json

```diff
@@ -8,7 +8,75 @@
     "moveInDate": "",
     "lastSignedIn": "",
     "actions": ["View Detail", "Re-invite", "Archive"],
-    "hasMovedIn": true
+    "hasMovedIn": true,
+    "viewDetail": {
+      "unit_details": [
+        {
+          "label": "Unit type",
+          "value": "Apartment"
+        },
+        {
+          "label": "Sub asset",
+          "value": "Nenya"
+        },
+        {
+          "label": "Apartment unit",
+          "value": "Nenya-apartment-Nenya"
+        }
+      ],
+      "tenant_details": [
+        {
+          "label": "Tenant Name",
+          "value": "Nivya Mathew"
+        },
+        {
+          "label": "Gender",
+          "value": "FEMALE"
+        },
+        {
+          "label": "DOB",
+          "value": "2025-01-01"
+        },
+        {
```

Comment:
When integrating the API, we will not receive data in the form of label and value. Instead, we will receive a JSON object like this:

{
    "name": "Nivya Mathew",
    "gender": "Female",
    "move_in_date": "12-01-2025",
    "phone_number": "+578908908",
    "apartment": "Nenya"
}
Please refactor the code accordingly to handle this structure.